### ISSUE TITLE
Asynchronous javascript calls not implemented

### ISSUE THREAD

**Issue by emanlove** [robotframework/SeleniumLibrary#20]
There is no rf-s2l keyword to call WebDriver's execute_async_script or exercise this asynchronous functionality. Note there is currently browser keywords for setting the selenium timeout which as far as I can see is the timeout only for asynchronous JavaScript calls.

**Comment by emanlove** [comment]
I've added tests and code for a rf-s2l keyword for executing asynchronous javascript calls within my [fix-s2l-timeout-bug branch](https://github.com/emanlove/robotframework-selenium2library/commit/39975f746fd73c8d5f4b2fc166d8559a3b1f3d3e).  The tests (test/acceptance/keywords/async_javascript.txt) mirror the same checks made by the selenium tests but using robotframework.

As I have not yet made the corrections for the set_selenium_timeout issue the test which expects a timeout exception is failing , as it should.  The test "Should Be Able To Execute Asynchronous Scripts" has not yet been implemented and has been set to fail as a reminder to update it.  Finally the test checking the passing of multiple arguments is failing.  When I implemented the asynchronous javascript functionality, I merely copied the synchronous javascript keyword which allows for multiple arguments all of which it assumes all is javascript code. But the selenium functions, both asynchronous and synchronous allows for additional arguments.  I might correct for that discrepancy later on.
